rolling-deploy.sh

```diff
@@ -126,3 +126,12 @@ cf rename "${APP_NAME}-${BUILD_NUMBER}" "${APP_NAME}"
 
 echo "Deleting the orphaned routes"
 cf delete-orphaned-routes -f
+
+# Enable PCF Autoscaling if autoscale.yml file is found
+curDir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
per pivotal's recommendation this script doesn't use manifests. The json file provides all the values and the pipelines control the variables.